cmd/crc/cmd/delete.go

```diff
@@ -51,7 +51,12 @@ func deleteMachine(client machine.Client, clearCache bool, cacheDir string, inte
 
 	yes := input.PromptUserForYesOrNo("Do you want to delete the OpenShift cluster", force)
 	if yes {
-		return true, client.Delete()
+		err := client.Delete()
+		// Delete the log file
+		if err := os.RemoveAll(constants.LogFilePath); err != nil {
```

Comment:
When running unit tests, this code will delete your log folder. Pass it as a variable.

Also, maybe create a method `logging.Destroy or Clean` in the logging package. It avoids the spread of this constant.